### ISSUE TITLE
Haunting Ground game issue

### ISSUE THREAD

**Issue by Fadi5555** [jpd002/Play-#329]
Hi there. Will the issue of this game when I try to select it from the menu it's just stay a couple of seconds then the play! App is crash (goes to home screen)
Please can you look at it to see what's wrong?

**Comment by jpd002** [comment]
Thanks for reporting the issue with the game. I'll get the game and take a look when I have some time.

**Comment by Fadi5555** [comment]
@jpd002 thanks for replay.
I guess there's something if you add it to play! It's gonna fix all games with a black screen or even crashing? Do you have any idea how to do that?

Other thing why the games on ios looks so pixels? Please try one on your idevice and you'll understand what i mean
For mire info please go to (ios port) there's some images to see what i mean.
Last thing play! On ios still not showing the images of cover games.

That's it thanks..

**Comment by jpd002** [comment]
It's not going to fix all games, but might fix some of them if this one is fixed. There's many things to be done and my time is limited, so, please be patient.

**Comment by Fadi5555** [comment]
@jpd002 thanks for replying.
Please also try to answer about my questions above. You just answer for one of them
I mean like pixels issue and other thing

Don't worry I'll be patient.
Thanks

**Comment by Fadi5555** [comment]
Anything about Haunting ground?

**Comment by Fadi5555** [comment]
If you receive it please make tests on it.
I'm excited to see it.
Thanks

**Comment by jpd002** [comment]
Got the game to boot, but it's not playable at the moment due to some issues with GS emulation.

![demento_1](https://cloud.githubusercontent.com/assets/6760433/11321897/c00927c6-909e-11e5-819b-60b492427bec.jpg)
![demento_2](https://cloud.githubusercontent.com/assets/6760433/11321898/c17b12e0-909e-11e5-8231-8df3e345238e.jpg)
![demento_3](https://cloud.githubusercontent.com/assets/6760433/11321899/c3478612-909e-11e5-9403-4dee03fe5f25.jpg)

**Comment by Fadi5555** [comment]
That' nice.
I'm really hope to fix this issues with GS emulation

**Comment by Fadi5555** [comment]
Do you have any plans to fix this graphic issue?

**Comment by jpd002** [comment]
Yes, but it will take time. This might require some changes to the way alpha blending is handled.

**Comment by Fadi5555** [comment]
Got the point.
I hope any fix come from you will effect other one who have same issue like when you fix resident evil 4 black screen it effects other games in my list were before on black screen. So about the graphic issue if you fix it i hope will fix other games as well like gta san.
Thanks again
